tests/DCPS/ReliableBestEffortReaders/TestMsg.idl

```diff
@@ -0,0 +1,7 @@
+@topic
```

Comment:
Does this test need its own IDL library?  Or can it use an existing one?

Reply:
I searched the idl's in all the DCPS tests. Two close ones are Partition/Test.idl and MultiDiscovery/TestMsg.idl. They have a long and a string members, but I would like one with a seq# separated from the key field, in case we need to setup different testing scenarios and still be able to check seq# separately.

Reply:
How about `tests/DCPS/ConsolidatedMessengerIdl/Messenger.idl`?

Reply:
Replaced TestMsg.idl with the shared ConsolidatedMessengerIdl/Messenger.idl. That's pretty cool. I'll check in the code shortly. I think we should use that shared idl in the transport level tests, too. Or create something like that.